server/src/com/thoughtworks/go/server/web/IgnoreResolver.java

```diff
@@ -76,6 +77,10 @@ private boolean isGoingToAbout(HttpServletRequest request) {
         return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/about");
     }
 
+    private boolean isGoingToPlugin(HttpServletRequest request) {
+        return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/plugins/interact");
```

Comment:
Ok, so it is ```/plugins/interact/{pluginId}/{requestName}```. Alright. There was a valid question about old-school controllers, from @ketan. Any reason we cannot move all of this to Rails?